rollup.config.js

```diff
@@ -10,12 +10,13 @@ const typescriptConfiguration = {
 };
 
 const iifeBundle = {
-  input: 'lib/qunit-dom-globals.ts',
+  input: 'lib/qunit-dom.ts',
 
   external: ['qunit'],
   plugins: [typescript(typescriptConfiguration)],
 
   output: {
+    name: 'QUnitDOM',
```

Comment:
Since we're now adding an export in what was previously `qunit-dom-globals.ts`, rollup expects you to provide an `output.name`, and will print a warning to that effect:

```shell
(!) If you do not supply "output.name", you may not be able to access the exports of an IIFE bundle.
```

Adding this name resolves that warning, and doesn't appear to have a detrimental effect.